llama_index/vector_stores/weaviate_utils.py

```diff
@@ -131,6 +131,11 @@ def _legacy_metadata_dict_to_node(entry: Dict[str, Any]) -> Tuple[dict, dict, di
     return extra_info, node_info, relationships
 
 
+def get_node_similarity(entry: Dict, similarity_key: str = "distance") -> float:
+    """Get node similarity."""
+    return entry["_additional"][similarity_key]
```

Comment:
Looks like we do a transformation in other impls, should probably do this here too? 

```python
similarity_score = 1.0 - math.exp(-distance)
```